fluentd/fluent.conf

```diff
@@ -111,9 +132,11 @@
   pos_file /var/log/node.log.pos
   tag system.var.log
   format multiline
-  format_firstline /^[A-Z][a-z]{2} [ |0-3]+[0-9] [0-2][0-9]:[0-5][0-9]:[0-5][0-9] /
-  format1 /^(?<time>[^ ]* [^ ]* [^ ]*) (?<host>[^ ]*) (?<ident>[a-zA-Z0-9_\/\.\-]*)(?:\[(?<pid>[0-9]+)\])?[^\:]*\: *(?<message>.*)$/
+  format_firstline /^[A-Z][a-z]{2} [ |0-3][0-9] [0-2][0-9]:[0-5][0-9]:[0-5][0-9] /
+  format1 /^(?<time>\S*\s+\S*\s+\S*) (?<host>\S*) (?<ident>[a-zA-Z0-9_\/\.\-]*)(?:\[(?<pid>[0-9]+)\])?[^\:]*\: *(?<message>.*)$/
```

Comment:
Oh, good point. They should be consistent.
If we're assured the line doesn't start with whitespace then we can leave it out. I just don't like having to check assumptions like that.